sql/guild_item_points.sql

```diff
@@ -23,1280 +23,913 @@ CREATE TABLE `guild_item_points` (
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8;
 
 -- ----------------------------
--- Records 
+-- Records
 -- ----------------------------
-INSERT INTO `guild_item_points` VALUES ('0', '2850', '9', '2850', '6240', '3');
-INSERT INTO `guild_item_points` VALUES ('0', '4288', '6', '1200', '5040', '2');
```

Comment:
for acts of bravery in the ongoing war against quoted non strings, I present you with this medal :medal_military:

Reply:
i rewrote this file from scratch :(

Reply:
ok fine, 2 medals. :medal_military: :medal_sports: